app/controllers/order_controller.rb

```diff
@@ -0,0 +1,31 @@
+class OrderController < ApplicationController
+
+  STANDARD = 2
+  PREMIUM = 5
+  VIP = 10
+
+  def total_price(order)
+    details = OrderDetail.where('order_id=' + order.id)
+    sum = 0
+    details.each { |detail|
+      book = Book.where('id=' + detail.book_id.to_s)
+      sum += book.price * detail.quantity
+    }
+    sum
+  end
+
+  def price(order)
+    user = User.where('id=' + order.user_id)
+    if user.level == 0
```

Comment:
Don't if in if. 
To easy, you can write: 

```
if user.level == 0
      discount = STANDARD
    elsif user.level == 1
        discount = PREMIUM
else
        discount = VIP
    end

Reply:
i got it